compiler/src/yul/mappers/functions.rs

```diff
@@ -63,6 +63,16 @@ pub fn func_def(
                     }
                 })
             }
+            FixedSize::String(string) => {
```

Comment:
see if these function mappings can be cleaned up. matching the exact return type is a weird. also, get rid of the memory freeing stuff